src/App.js

```diff
@@ -1,13 +1,47 @@
-import React from "react";
+//api_key=1F5F9mMau9iKruv7BeHJOWu1C34yR20vUAbzJYiN
+
+import React, { useEffect, useState } from "react";
 import "./App.css";
+import axios from "axios";
+import { Card } from "./components/CardComponent/Card";
 
 function App() {
+  const [nasaInfo, setNasaInfo] = useState([]);
+
+  useEffect(() => {
+    for(let i = 1; i <10; i++){
+      axios
+      .get(
+        `https://api.nasa.gov/planetary/apod?api_key=1F5F9mMau9iKruv7BeHJOWu1C34yR20vUAbzJYiN&date=2012-03-${i}`
+      )
+      .then(res => {
+        setNasaInfo(nasaInfo => [...nasaInfo, res.data]);
```

Comment:
This is a great alternative to setting state instead of using the concat method.